packages/nx-plugin/src/generators/configure-playwright/files/src/index.spec.tsx.template

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
```

Comment:
this should be part of component separate generator, similar to https://nx.dev/nx-api/react/generators/component-cypress-spec but for playwright ofc.

can you create feature request on nx repo to add component testing capabilities to core nx playwright plugin ?

Reply:
Here's the feature request for nx https://github.com/nrwl/nx/issues/21594

Reply:
Ok, now we got 2 generators and 1 executor:

1. executor:
    * `nx-plugin:playwright`
2. generators:
    * `playwright-component-configuration` (as per https://nx.dev/nx-api/react/generators/cypress-component-configuration)
    * `component-playwright-spec` (as per https://nx.dev/nx-api/react/generators/component-cypress-spec)